src/mapping/config/MappingConfiguration.hpp

```diff
@@ -169,6 +185,22 @@ class MappingConfiguration : public xml::XMLTag::Listener {
   const std::string ATTR_SHAPE_PARAM    = "shape-parameter";
   const std::string ATTR_SUPPORT_RADIUS = "support-radius";
 
+  // For iterative RBFs using Ginkgo
+  const std::string SUBTAG_EXECUTOR = "executor";
+  const std::string EXECUTOR_CPU    = "cpu";
+  const std::string EXECUTOR_CUDA   = "cuda";
+  const std::string EXECUTOR_HIP    = "hip";
+  const std::string EXECUTOR_OMP    = "openmp";
+
+  const std::string ATTR_DEVICE_ID = "gpu-device-id";
+  const std::string ATTR_N_THREADS = "n-threads";
+  // const std::string ATTR_ENABLE_UNIFIED_MEMORY = "enable-unified-memory";
+  // const std::string ATTR_SOLVER                = "solver";
+  // const std::string ATTR_USE_PRECONDITIONER    = "use-preconditioner";
+  // const std::string ATTR_PRECONDITIONER        = "preconditioner";
+  // const std::string ATTR_JACOBI_BLOCK_SIZE     = "jacobi-block-size";
+  // const std::string ATTR_MAX_ITERATIONS        = "max-iterations";
```

Comment:
still needed?

Reply:
It would be good to keep an overview of available options for the moment, yes.